### PR TITLE
Create troubleshooting guides for NVIDIA Resnet example

### PR DESCRIPTION
**Pull request by vinhngx**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1388)

### CODE REVIEW

**Comment by vinhngx** [comment]
I signed it!

**Comment by Ark-kun** [comment]
This troubleshooting guide looks generic and not specific to Resnet sample.

**Comment by hongye-sun** [comment]
I agree with Alexey. This is a very useful guide to help user to debug pipeline errors. Sarah, do you think we can add it as one part of KFP documentation?

File: samples/nvidia-resnet/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Pipelines with error status can be inspected with:
+
+`kubectl logs   resnet-cifar10-pipeline-t2kv2-4047898736 --namespace kubeflow  --all-containers`
+
+To continuously monitor a pipeline:
+
+`kubectl logs -f   resnet-cifar10-pipeline-t2kv2-4047898736 --namespace kubeflow  --all-containers`
+
+## 1. Kubeflow ambassador stuck in a crash loop.
```
Comment (hongye-sun) [inline samples/nvidia-resnet/troubleshooting.md]: After this line, it's for kubeflow troubleshooting. Before this line, they can be KFP tsg.

**Comment by sarahmaddox** [comment]
> I agree with Alexey. This is a very useful guide to help user to debug pipeline errors. Sarah, do you think we can add it as one part of KFP documentation?

Sure! Here's the generic Kubeflow troubleshooting guide:
https://www.kubeflow.org/docs/other-guides/troubleshooting/

We don't yet have a Pipelines troubleshooting guide. If much of the content is pipelines-specific, you can create a troubleshooting guide in this section:
https://www.kubeflow.org/docs/pipelines/

There's a mention of Envoy crash-looping (GCP-specific) here:
https://www.kubeflow.org/docs/gke/troubleshooting-gke/#envoy-pods-crash-looping-root-cause-is-backend-quota-exceeded

**Pull request closed without merging**